hasktorch/hasktorch.cabal

```diff
@@ -120,6 +122,9 @@ test-suite spec
                     , QuickCheck
                     , directory
                     , JuicyPixels
+                    , data-default
```

Comment:
It is not necessary to use it.
I think users are familer with the package, because dozens of packages uses it.
https://www.stackage.org/lts-14.5/package/data-default-0.7.1.1/revdeps